src/networkextension.cs

```diff
@@ -2104,4 +2134,144 @@ interface NETransparentProxyNetworkSettings {
 		[NullAllowed, Export ("excludedNetworkRules", ArgumentSemantic.Copy)]
 		NENetworkRule[] ExcludedNetworkRules { get; set; }
 	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	interface NEAppPushManager {
+		[Export ("matchSSIDs", ArgumentSemantic.Copy)]
+		string[] MatchSsIds { get; set; }
```

Comment:
I believe SSID is one big acronym, so:

```suggestion
		string[] MatchSsids { get; set; }
```

It's also what we did here: https://github.com/xamarin/xamarin-macios/blob/98a4c2bd609df1718f536b70004dd2c5dea3cf4c/src/networkextension.cs#L1990